Model.cpp

```diff
@@ -129,6 +137,15 @@ int Model::insertBatch(vector<Model*> models, int batchsize, Mode mode)
                         conn.setString(paramNum, i->getValue());
                     }
                 }
+                else if(f->getType() == BOOLEAN)
```

Comment:
Just a side note, we should probably convert this if statement to a switch at some point in the future. Long term, there is a repeating pattern here that we could probably refactor.